src/commands/BeautifyCommand.ts

```diff
@@ -143,6 +142,40 @@ export class BeautifyCommand extends BaseCommand {
       });
     });
   }
+
+  private validateLanguage(
+    language: string | undefined,
+    unibeautify: Unibeautify
+  ): Promise<void> {
+    if (!language) {
+      const error = new Error("A language is required.");
+      return this.handleError(error, 1);
+    }
+    const langs = unibeautify.findLanguages({ name: language });
+    if (langs.length === 0) {
+      const allLanguages = getAllLanguages();
+      const distances: number[] = allLanguages.map(lang =>
+        levenshtein.get(lang.toLowerCase(), language.toLowerCase())
+      );
+      const bestDistance: number = Math.min(...distances);
+      const distanceThreshold = 2;
+      const bestMatchLanguages = allLanguages.filter(
+        (lang, index) =>
+          distances[index] <= Math.min(distanceThreshold, bestDistance)
+      );
+      if (bestMatchLanguages.length > 0) {
+        const limit = 3;
+        const error = new Error(
+          `Language '${language}' was not found. Did you mean:\n\n${bestMatchLanguages
```

Comment:
I don't think I like the double `\n` here.

I just noticed the snapshot at https://github.com/Unibeautify/unibeautify-cli/pull/116/files#diff-79f3cbac345f885b9f1a9467fa09ae86R55

```suggestion
          `Language '${language}' was not found. Did you mean:\n${bestMatchLanguages
```